templates/d1/seed.ts

```diff
@@ -57,7 +63,11 @@ function getLocalD1Db() {
   const client = createClient({
     url: `file:${pathToDb}`,
```

Comment:
there's a fix to the local path to the DB i've been meaning to get in for a while. will put up a separate PR once this is merged: https://github.com/fiberplane/create-honc-app/issues/82